src/main/webapp/WEB-INF/views/index.jsp

```diff
@@ -0,0 +1,16 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: Tsven
+  Date: 20.01.2021
+  Time: 18:29
+  To change this template use File | Settings | File Templates.
+--%>
```

Comment:
Remove autogenerated comments from index.jsp. 
Before submitting solution make sure you checked it first with checklist

Reply:
done